automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -12,12 +11,18 @@ import org.scalatest.selenium.Page
 class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodConfigNamespace: String, val methodConfigName: String)(implicit webDriver: WebDriver)
   extends WorkspacePage(namespace, name) with Page with PageUtil[WorkspaceMethodConfigDetailsPage] {
 
+  private val tab = TabBar()
+  private val methodConfigListpage = new WorkspaceMethodConfigListPage(namespace, name)
+
   override def awaitReady(): Unit = {
-    await condition isLoaded
-    await spinner "Checking permissions..."
+    await condition {
+      tab.isTabOpen("Method Configurations")
+      isLoaded
+    }
+    await.spinner("Checking permissions...",15) // brittle. won't work if text changes
   }
 
-  override val url: String = s"${Config.FireCloud.baseUrl}#workspaces/$namespace/$name/method-configs/$methodConfigNamespace/$methodConfigName"
+  override val url: String = s"${methodConfigListpage.url}/$methodConfigNamespace/$methodConfigName"
```

Comment:
I don't like the idea of creating a separate page just to borrow its url.

Reply:
Yeah, it's not ideal. You could extract a function into a companion object:
```
object WorkspaceMethodConfigListPage {
  def url(namespace: String, name: String): String = s"${Config.FireCloud.baseUrl}#workspaces/$namespace/$name/method-configs"
}
```
to enable reuse of that logic while avoiding extra object instantiation.